### PR TITLE
Implements necessary changes for #50 

### PR DESCRIPTION
**Pull request by Umashankar5995**
### SUMMARY

Implements necessary changes for #50 

*Reviewers:* @numere-org/maintainers

### IMPLEMENTATION
* Implementation: (*Describe, what you've changed*) 
* Implementation test: (*Describe the type of test, which you performed, and if it was successful*)

### DOCUMENTATION
* [x] ChangesLog updated
* [x] Code changes commented
* **Documentation articles:**
    * [ ] corresponding documentation articles updated
    * [ ] new documentation articles created
    * [x] not needed
* **Language files:**
    * [x] corresponding language files updated
    * [ ] not needed

----------------

### TESTS BY REVIEWERS
* [ ] Added to the automatic SW tests
* [x] Tested manually

### CODE REVIEW

File: gui/terminal/terminal.cpp [hunk 1]
```diff
@@ -510,7 +510,7 @@ wxThread::ExitCode NumeReTerminal::Entry()
 
 		// During idle times so that these tasks don't interfere with the main evaluation routine
 		// do the following:
-		if (time(0) - _kernel.getLastSavedTime() >= _kernel.getAutosaveInterval())
+		if (_kernel.getAutosaveInterval() > 0 && (time(0)-_kernel.getLastSavedTime()>=_kernel.getAutosaveInterval()))
```
Comment (numeredev) [inline gui/terminal/terminal.cpp:513]:
```suggestion
		if (_kernel.getAutosaveInterval() > 0 && (time(0) - _kernel.getLastSavedTime() >= _kernel.getAutosaveInterval()))

```
We prefer, if there's whitespace around the operators

**Review by numeredev — CHANGES_REQUESTED** [review]
Please correct the styling and add the missing changes log as well.

**Comment by Umashankar5995** [comment]
if (_kernel.getAutosaveInterval() > 0 && (time(0) - _kernel.getLastSavedTime() >= _kernel.getAutosaveInterval()))

File: Debug/Profiling/NumeRe.res
Comment (numeredev) [inline Debug/Profiling/NumeRe.res:1]: Please remove this file. It is auto-created and binary

**Review by numeredev — APPROVED** [review]
Changes accepted. Will be merged

**Pull request merged**